dispatcher.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"io"
+	"log"
+	"net"
+)
+
+func game_tx(gameOutChannel chan InFrame, gameConn net.Conn) {
+	for {
+		log.Println("game_tx")
+		// Take the outbound frame to the game
+		frame := <-gameOutChannel
+
+		// Forward it to the game server
+		numWritten, err := gameConn.Write(frame.data) // TODO: Convert the entire frame to []byte and send it all
+		// TODO: Or perhaps these frames should be a different type altogether
+		if err != nil || numWritten < len(frame.data) { // See above TODO for len check
```

Comment:
Hmm... I think these two cases are different, and needs to be handled diffrently